ui/src/components/SampleView/SampleControls.js

```diff
@@ -263,41 +252,36 @@ export default class SampleControls extends React.Component {
           rootClose
           placement="bottom"
           overlay={
-            <span className="slider-overlay" style={{ marginTop: '8px' }}>
-              {zoom_motor.limits[0]}
+            <div className={styles.overlay}>
               <input
-                className="bar"
+                className={styles.zoomSlider}
                 type="range"
-                id="zoom-control"
-                min={zoom_motor.limits[0] - 1}
+                min={zoom_motor.limits[0]}
                 max={zoom_motor.limits[1]}
-                step="1"
                 value={zoom_motor.commands.indexOf(zoom_motor.value)}
                 disabled={zoom_motor.state !== MOTOR_STATE.READY}
-                onMouseUp={this.setZoom}
+                onMouseUp={(e) => {
+                  this.props.sendSetAttribute(
+                    'diffractometer.zoom',
+                    zoom_motor.commands[Number.parseFloat(e.target.value)],
+                  );
+                }}
                 onChange={(e) => {
                   this.props.setBeamlineAttribute(
                     'diffractometer.zoom',
-                    zoom_motor.commands[Number.parseFloat(e.target.value) - 1],
+                    zoom_motor.commands[Number.parseFloat(e.target.value)],
                   );
                 }}
                 list="volsettings"
                 name="zoomSlider"
               />
-              {zoom_motor.limits[1]}
 
               <datalist id="volsettings">
-                {[
-                  ...Array.from({
-                    length: zoom_motor.limits[1] - zoom_motor.limits[0],
-                  }).keys(),
-                ].map((i) => (
-                  <option key={`volsettings-${i}`}>
-                    {zoom_motor.limits[0] + i}
```

Comment:
`zoom_motor.limits[0]` seems to always be 0, but maybe I missed something?

Reply:
This depends on the hardware.